SPECS/grpc/grpc.spec

```
@@ -78,16 +78,30 @@ find %{buildroot} -name '*.cmake' -delete
 %{_datadir}/grpc/roots.pem
 
 %files devel
-%{_includedir}/*
-%{_libdir}/*.so
-%{_lib64dir}/*.so
+%{_includedir}/grpc
+%{_includedir}/grpc++
+%{_includedir}/grpcpp
+%{_libdir}/libgpr.so
+%{_libdir}/libgrpc++.so
+%{_libdir}/libgrpc++_alts.so
+%{_libdir}/libgrpc++_error_details.so
+%{_libdir}/libgrpc++_reflection.so
+%{_libdir}/libgrpc++_unsecure.so
+%{_libdir}/libgrpc.so
+%{_libdir}/libgrpc_plugin_support.so
+%{_libdir}/libgrpc_unsecure.so
+%{_libdir}/libgrpcpp_channelz.so
+%{_libdir}/libupb.so
```

Are you sure this builds? The build usually fails, if there are any installed but unpackaged files (now the `%{_includedir}/re2` is unpackaged for instance). This can be changed with `%define _unpackaged_files_terminate_build 0` but I don't see it here and I think it's safer to explicitly say, which files we're including/excluding anyway.

```suggestion
%{_includedir}/grpc
%{_includedir}/grpc++
%{_includedir}/grpcpp
%exclude %{_includedir}/re2

%{_libdir}/libgpr.so
%{_libdir}/libgrpc++.so
%{_libdir}/libgrpc++_alts.so
%{_libdir}/libgrpc++_error_details.so
%{_libdir}/libgrpc++_reflection.so
%{_libdir}/libgrpc++_unsecure.so
%{_libdir}/libgrpc.so
%{_libdir}/libgrpc_plugin_support.so
%{_libdir}/libgrpc_unsecure.so
%{_libdir}/libgrpcpp_channelz.so
%{_libdir}/libupb.so
%exclude %{_libdir}/libabsl*.so
%exclude %{_libdir}/libaddress_sorting.so
%exclude %{_libdir}/libre2.so
%exclude %{_libdir}/libupb.so

%exclude %{_lib64dir}/libre2.so
```

---

I did find some errors in local validation and will update here with the fix